### PR TITLE
be more clear about what is a translation

### PR DESCRIPTION
**Pull request by i30817**

### CODE REVIEW

File: metadat/hacks/Sega - Saturn.dat [hunk 1]
```diff
@@ -22,23 +22,23 @@ game (
 )
 
 game (
-	name "Asuka 120% Limit Over (menu translation)"
+	name "Asuka 120% Limit Over (english translation)"
 	description "Asuka 120% Limit Over by CafeBreak Project version 12/31 + Menu Translation by Ms. Tea version 1.0"
 	rom ( name "Asuka 120% Limited - Burning Fest. Limited (Japan) (Track 01).bin" size 62095152 crc bd23db54 md5 4bcce0bfcc29cb358f7c2e2c8682f1a9 sha1 b9723e2277933df5fba915bcf160253952ba5fdd )
```
[the remark this answers is not in the document]
Comment (i30817) [reply metadat/hacks/Sega - Saturn.dat]: And replace by what? 'Asuka 120 Percent Limit Over'?

File: metadat/hacks/Sega - Saturn.dat [hunk 2]
```diff
@@ -4,7 +4,7 @@ clrmamepro (
 )
 
 game (
-	name "Dragon Force II: The Godforsaken Land"
+	name "Dragon Force II: The Godforsaken Land (english translation)"
```
[the remark this answers is not in the document]
Comment (i30817) [reply metadat/hacks/Sega - Saturn.dat]: What do we do where the game wasn't a published dump? 

Ie for example, Asuka 120 over limit is a hack of a game that exists but was never published by redump for obvious reasons, and 'Shining Force III Collection Deluxe' is a hack joining more than one game on a single cd. 

Or does that not matter, but the that 'ultimate origin' of the game was Japan even if it's a translation of a hack or several games on a cd?

**Comment by i30817** [comment]
Also i have a question about recognizing 'illegal hacks' @RobLoach 

I have one hack that pretty much turns the No-intro version of Tsumi to Batsu - Hoshi no Keishousha (Japan) for the N64 to Sin and Punishment - Successor of the Earth for the virtual console.

This (soft) patch was made by extracting the VC rom and making a patch after byte swapping to the right byte order, and is obviously illegal to distribute.

Anyway i expected that virtual console roms would be already represented by no-intro but this doesn't seem to be the case. Can i add this to the hacks of Nintendo 64 and what format should its name follow?

File: metadat/hacks/Sega - Saturn.dat (hunk 1 above)
[the remark this answers is not in the document]
Comment (RobLoach) [reply metadat/hacks/Sega - Saturn.dat]: "Percent" would work.

**Comment by RobLoach** [comment]
You can distribute the hashes, just not the binary.

https://github.com/libretro/libretro-database/blob/master/metadat/hacks/Nintendo%20-%20Nintendo%2064.dat would be a good place for the N64 ones.

**Comment by RobLoach** [comment]
Looks good!

**Comment by i30817** [comment]
Well there are some problems here, including a confusion between using 3 digits or two for the language code. I'll probably fix it later to use 2 digits.

Right now i'm doing a script that will allow me to scan my whole collection of nes+snes+megadrive and do those all at once in different PRs.

I needs a file with the 'vVersion-number readme.txt' and a link to romhacking.net inside and then it'll scrap out the 'real' hack name, the author, language and the patch version (to compare to local version and give a warning), then it patches the roms with the softpatch on the dir, then creates dat entries to file. That should allow me to get all of them or nearly.

It only doesn't work too well with the description (becomes generic), and two or more hacks applied, the md5sum is correct but only the first hack is shown. I hope that's acceptable.

**Pull request merged**